node/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -672,7 +674,8 @@ pub fn get_transaction(hash: Result<(Hash,), jsonrpc_core::Error>) -> JsonRpcRes
             .then(move |res| match res {
                 Ok(Ok((transaction, pointer_to_block))) => {
                     let output = GetTransactionOutput {
-                        transaction,
+                        transaction: transaction.clone(),
+                        size: transaction.size(),
```

Comment:
Usually this kind of clones can be avoided by changing the order of operations. In this case you can do `let size = transaction.size();` And then this simply becomes:
```suggestion
                        transaction,
                        size,
```